File: Python/Program/HardWare.py
class HardWare:
    def __init__(self, name="", brand="", price=0.0, warranty=0):
        self.name = name
        self.brand = brand
        self.price = price
        self.warranty = warranty

    def set_name(self, name):
        self.name = name

    def set_brand(self, brand):
        self.brand = brand

    def set_price(self, price):
        self.price = price

    def set_warranty(self, warranty):
        self.warranty = warranty

    def get_name(self):
        return self.name

    def get_brand(self):
        return self.brand

    def get_price(self):
        return self.price

    def get_warranty(self):
        return self.warranty

File: Java/Program/HardWare.java
public class HardWare {
    protected String name;
    protected String brand;
    protected double price;
    protected int warranty;

    public HardWare() {}

    public HardWare(String name, String brand, double price, int warranty) {
        this.name = name;
        this.brand = brand;
        this.price = price;
        this.warranty = warranty;
    }

    public void setName(String name) { this.name = name; }
    public void setBrand(String brand) { this.brand = brand; }
    public void setPrice(double price) { this.price = price; }
    public void setWarranty(int warranty) { this.warranty = warranty; }

    public String getName() { return name; }
    public String getBrand() { return brand; }
    public double getPrice() { return price; }
    public int getWarranty() { return warranty; }
}
